apps/web/.env.example

```
@@ -0,0 +1,8 @@
+# Base variables (shared across all apps)
+DATABASE_URL=
+NEXT_PUBLIC_CORS_ORIGIN=http://localhost:3002
+NEXT_PUBLIC_NODE_ENV=local
+
+# Web app specific variables
+NEXT_PUBLIC_WEB_URL=http://localhost:3001
+NEXT_PUBLIC_DASHBOARD_URL=http://localhost:3003
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)